src/constant_texts.c

```diff
@@ -59,6 +59,12 @@
 #include "ui_menu.h"
 #include "ui_text_slideshow.h"
 
+// Onboarding text
+const char *ui_text_onboarding[] = {
```

Comment:
this will enable compiler warnings

```suggestion
const char *ui_text_onboarding[NUMBER_OF_SLIDESHOW_SCREENS_ONBOARDING] = {
```